traffic_ops/traffic_ops_golang/login/register.go

```diff
@@ -164,6 +164,15 @@ func RegisterUser(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, tx, http.StatusBadRequest, userErr, nil)
 		return
 	}
+	// ToDo: uncomment this once the perm based roles and config options are implemented
+	//var reqV50 tc.UserRegistrationRequestV50
+	//if inf.Version.Major >= 5 {
+	//	if userErr = api.Parse(r.Body, tx, &reqV50); userErr != nil {
+	//		api.HandleErr(w, r, tx, http.StatusBadRequest, userErr, nil)
+	//		return
+	//	}
+	//	tenantID = reqV50.TenantID
+	//}
```

Comment:
Why can't we do this now? Or if you want to do it in another PR, why leave the comment in?